src/media/writer.rs

```diff
@@ -120,7 +123,7 @@ impl<'a> Writer<'a> {
     /// a=rtcp-fb:96 nack pli
     /// ```
     pub fn is_request_keyframe_possible(&self, kind: KeyframeRequestKind) -> bool {
-        media_by_mid(&self.session.medias, self.mid).is_request_keyframe_possible(kind)
```

Comment:
Should we reject sdps that violate these feedbacks being tied to the pt?

```
m=video ...
a=rtpmap:96 VP8/90000
a=rtcp-fb:96 nack
m=video ...
a=rtpmap:96 VP8/90000
// No NACK
```

https://datatracker.ietf.org/doc/html/draft-ietf-mmusic-sdp-mux-attributes-17#section-4.7 says it needs to be identical

Reply:
There's certainly more validation that should be done of the remote SDP. I think we start a TODO list in change/sdp.rs, cause I know there is another one I can't quite remember right now.